README.md

```diff
@@ -16,113 +16,89 @@ dependencies {
 }
 ```
 
-Then, declare `BrowserSwitchActivity` in your `AndroidManifest.xml`:
+To preview the latest work in progress builds, add the following SNAPSHOT dependency in your `build.gradle`:
+
+```groovy
+dependencies {
+  implementation 'com.braintreepayments.api:browser-switch:2.3.3-SNAPSHOT'
```

Comment:
NIT - maybe let's not tie this to a version to future-proof it.

Reply:
Nevermind, just saw the `task changeREADMEVersion` below! 👏

Reply:
This actually gets updated by the GitHub actions script to be the most recent.